logprep/processor/generic_resolver/rule.py

```diff
@@ -98,16 +117,7 @@ class Config(FieldManagerRule.Config):
             ]
         )
         """Mapping in form of :code:`{SOURCE_FIELD: DESTINATION_FIELD}`"""
-        resolve_list: dict = field(
-            validator=[
-                validators.instance_of(dict),
-                validators.deep_mapping(
-                    key_validator=validators.instance_of(str),
-                    value_validator=validators.instance_of(str),
-                ),
-            ],
-            factory=dict,
-        )
+        resolve_list: dict = field(validator=[validators.instance_of(dict)], factory=dict)
```

Comment:
Why did you simplify the validator?

Reply:
I wanted the ability to add non-string values like dictionaries.
The only thing preventing that is the validator.

Reply:
ah okay, I didn't find that anywhere that's why I was wondering. Do you want to add a small test for this then, just so we fix this behavior and nobody will change it again in the future.

Reply:
Sure, I have added a test.